### PR TITLE
Update ko translations

### PR DESCRIPTION
**Pull request by derornos**

### CODE REVIEW

**Comment by derornos** [comment]
Hi, @zzorba 

please check a line: 
https://github.com/zzorba/arkham-cards-data/blob/2a946a57904a7d89d61f434df81a7a2c2bd9efb3/i18n/ko/campaigns/zcp/campaign.po#L43

**Comment by zzorba** [comment]
Yeah, so that line is meant to handle the special rules of the Call of the Plaguebearer, where there are sometimes sections you 'are not instructed' to read, but if you inadvertently read them you are supposed to do their effects.

To make this work in the app, I have a button that you can press with that text, to indicate that you should apply the effects you just read to your campaign.

Does that make sense?

File: i18n/ko/campaigns/tcu/disappearance_at_the_twilight_estate.po [hunk 1]
```diff
@@ -448,3 +448,6 @@ msgstr "'페니의 운명'을 기록하기"
 
 msgid "Would you like to skip the prologue by drawing tarot cards?"
 msgstr "타로 카드 뽑기를 통해 서막을 건너뛰기 하시겠습니까? (‘돌아온 황혼회 저택에서의 실종’에서 추가된 선택 규칙입니다.)"
+
+msgid "Do you want to use the app to draw random tarot cards?"
+msgstr "앱을 통해 무작위로 타로 카드를 뽑으시겠습니까?"
```
Comment (zzorba) [inline i18n/ko/campaigns/tcu/disappearance_at_the_twilight_estate.po:453]: Thanks for flagging this, it actually goes in other project but I'll make sure this translation makes it over there.

**Comment by derornos** [comment]
> Yeah, so that line is meant to handle the special rules of the Call of the Plaguebearer, where there are sometimes sections you 'are not instructed' to read, but if you inadvertently read them you are supposed to do their effects.
> 
> To make this work in the app, I have a button that you can press with that text, to indicate that you should apply the effects you just read to your campaign.
> 
> Does that make sense?

Um... What I meant was that this translation didn't seem to work.

SmartSelect_20221026_063130_ArkhamCards

**Comment by zzorba** [comment]
Got it, yeah it was missing from the script.  Fixed in next release.

**Comment by derornos** [comment]
oh, thanks XD

**Pull request merged**